go/vt/sqlparser/parse_test.go

```diff
@@ -39,6 +39,24 @@ type parseTest struct {
 
 var (
 	validSQL = []parseTest{
+		{
+			input:  "select * from my_table_function()",
+			output: "select * from my_table_function()",
+		},
+		{
+			input:  "select * from my_table_function('foo')",
+			output: "select * from my_table_function('foo')",
+		},
+		{
+			input:  "select * from my_table_function('foo', 'bar')",
+			output: "select * from my_table_function('foo', 'bar')",
+		},
+		// TODO: We should be able to alias table functions, but that will require additional changes
```

Comment:
No commented out tests. Put them in their own test block and run them with t.Skip() instead

Reply:
Good rule. I went ahead and pulled this out. It'll be super easy to add back when we add alias support.